techeria_controller/techeria_app/views.py

```diff
@@ -60,6 +60,21 @@ def cart(request):
     context = {'order':order}
     return render(request, 'cart.html',context)
 
+
+def remove_cart(request,pk):
+    p = Products.objects.get(id=pk)
+    try:
+	    buyer = request.user.buyer
+    except:
+	    device = request.COOKIES['device']
+	    buyer, created = BuyerModel.objects.get_or_create(device=device)
```

Comment:
it's good that you have used **get_or_create** to check if the item you want to remove from the cart is found or not